shared/src/business/useCases/prioritizeCaseInteractor.old.test.js

```diff
@@ -0,0 +1,117 @@
+const { applicationContext } = require('../test/createTestApplicationContext');
```

Comment:
There's no new prioritizeCaseinteractor test file included in this PR - seems like that should be added with an updated test for the new if statement above.

Reply:
New test included to cover this code.